native-schema-registry/c/include/read_only_byte_array.h

```diff
@@ -0,0 +1,35 @@
+#ifndef READ_ONLY_BYTE_ARRAY_H
+#define READ_ONLY_BYTE_ARRAY_H
+#include <stdlib.h>
+#include "glue_schema_registry_error.h"
+
+typedef struct read_only_byte_array {
```

Comment:
I don't see what in the type system is making this read only?

Reply:
My intention is to create a structure that doesn't manage the memory of underlying array and provides a way to read it.